lib/rules/jsx-curly-brace-presence.js

```diff
@@ -103,34 +127,42 @@ module.exports = {
         node: literalNode,
         message: 'Need to wrap this literal in a JSX expression.',
         fix: function(fixer) {
+          // Leave it to the author to fix as it can be fixed by either a
+          // real character or unicode
+          if (containsHTMLEntity(literalNode.raw)) {
```

Comment:
We could use `String.fromCharCode` here to fix but based on your [comment](https://github.com/yannickcr/eslint-plugin-react/issues/1449#issuecomment-332750344), it may not be what the author has in mind.